app/views/admin/configs/_system_cohorts.haml

```diff
@@ -13,5 +13,6 @@
     = f.input :chronic_cohort, label: 'Chronic', hint: 'The chronic cohort includes all clients who are currently chronically homeless with open enrollment in ES, SO, TH, SH and no move-in date in a current PH project.'
     = f.input :adult_and_child_cohort, label: 'Adult and Child', hint: 'The adult and child cohort includes all clients in an adult and child household with open enrollment in ES, SO, TH, SH and no move-in date in a current PH project.'
     = f.input :adult_only_cohort, label: 'Adult Only', hint: 'The adult only cohort includes all clients in a household with only adults with open enrollment in ES, SO, TH, SH and no move-in date in a current PH project.'
+    = f.input :chronic_individual_cohort, label: 'Chronic Individual', hint: 'The chronic individidual cohort includes all unaccompanied clients who are currently chronically homeless with open enrollment in ES, SO, TH, SH and no move-in date in a current PH project.'
```

Comment:
This should be all "Adult only households" or maybe we want to go with Adult only or child only.  Not sure on that.